sys-power/thermald/thermald-2.4.5-r1.ebuild

```diff
@@ -22,16 +22,16 @@ RDEPEND="
 	sys-power/upower
 	sys-apps/dbus:="
 DEPEND="${RDEPEND}
-	dev-util/gtk-doc
+	doc? ( dev-util/gtk-doc )
```

Comment:
sorry, I didn't review the patch carefully..

you should introduce a "--enable-doc" option for upstream, and also avoid the conditional patch here